### PR TITLE
More detailed setup instructions

### PR DESCRIPTION
**Pull request by recardo-7**
Adds new sections to and fixes an oversight in INSTALLATION.md
Also show the user the button combo for toggling widescreen and add a warning for F3DEX3 compatibilty
For new users to get started more easily

### CODE REVIEW

File: INSTALLATION.md [hunk 1]
```diff
@@ -102,12 +102,36 @@ If all goes well, a new ROM should be built at `build/hackeroot-mq/hackeroot-mq.
...
 ```
 
 The patch will be in the ``build/hackeroot-mq/`` folder.
+
+#### 7. Toggle HackerOoT Features
+
+You can customize which HackerOoT features you want in your project and which not in the `include/config/` folder.
+Just define the features as `true` or `false` or, if necessary, put in the value you want (e.g. Text speed).
+
+**TIP:** if you are using VS Code, it might show you errors in your files with red squiglly lines. Those are false alarms, to get rid of them, you can place [THIS FILE](https://gist.github.com/krm01/7345ca80e0e7f6b083a0ea4e85515d4b) by [@kentonm](https://www.github.com/krm01) in your `.vscode` folder (You might need to create it first).
```
Comment (Yanis42) [inline INSTALLATION.md:137]: hackeroot has updated vscode instructions inside docs/vscode.md, also this file explains how to setup the Ares GDB stuff on vscode

**Review by Yanis42 — COMMENTED** [review]
nice!

File: INSTALLATION.md (hunk 1 above)
Comment (recardo-7) [reply INSTALLATION.md:137]: I'd link that file here then because it's a bit hard to find right now imo

Comment (Yanis42) [reply INSTALLATION.md:137]: yeah I agree it's kinda hidden, would be nice to have something written about it in the readme

File: INSTALLATION.md [hunk 2]
```diff
@@ -102,12 +102,36 @@ If all goes well, a new ROM should be built at `build/hackeroot-mq/hackeroot-mq.
...
 ```
 
 The patch will be in the ``build/hackeroot-mq/`` folder.
+
+#### 7. Toggle HackerOoT Features
+
+You can customize which HackerOoT features you want in your project and which not in the `include/config/` folder.
+Just define the features as `true` or `false` or, if necessary, put in the value you want (e.g. Text speed).
+
+**TIP:** if you are using VS Code, look at [vscode.md](docs/vscode.md) for useful costumization that make working with this project easier.
```
Comment (Yanis42) [inline INSTALLATION.md:137]: "costumization"

Comment (recardo-7) [reply INSTALLATION.md:137]: i meant like costumizing vscode for the project, couldnt think of a better word

Comment (Yanis42) [reply INSTALLATION.md:137]: isn't it "customization"? I thought you did a small typo here

Comment (recardo-7) [reply INSTALLATION.md]: ah, didn't even notice that
fixed now

**Review by Yanis42 — APPROVED** [review]

**Pull request merged**